mlir/test/Conversion/VectorToLLVM/vector-to-llvm.mlir

```diff
@@ -1737,3 +1737,40 @@ func.func @step() -> vector<4xindex> {
   %0 = vector.step : vector<4xindex>
   return %0 : vector<4xindex>
 }
+
+
+// -----
+
+//===----------------------------------------------------------------------===//
+// vector.from_elements
+//===----------------------------------------------------------------------===//
+
+// NOTE: For now, we unroll multi-dimensional from_elements ops with pattern `UnrollFromElements`
```

Comment:
```suggestion
// NOTE: We unroll multi-dimensional from_elements ops with pattern `UnrollFromElements`
```
Unless there is a proposal for this to be done differently, which I might have missed